bundles/CoreBundle/EventListener/Frontend/ElementListener.php

```diff
@@ -143,7 +143,7 @@ protected function handleVersion(Request $request, Document $document)
     {
         if ($request->get('v')) {
             if ($version = Version::getById($request->get('v'))) {
-                if ($version->getPublic()) {
+                if ($version->getPublic() || $request->get('pimcore_version')) {
```

Comment:
This would allow unauthenticated users to request any version, no matter if it was marked as public or not and could be seen as a data breach. 🤔